gooddata-flight-server/gooddata_flight_server/tasks/task.py

```diff
@@ -104,10 +105,8 @@ def cancel(self) -> bool:
             self._cancelled = True
 
             if first_cancel:
-                try:
+                with contextlib.suppress(Exception):
```

Comment:
This will make the happy path slower every time.

Reply:
I added the relevant rule SIM105 to ignore.